javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,104 @@
+// Flight booking fullname function
+// Create a function called getFullname that returns a fullname. It should have two parameters: firstname and surname.
+function getFullName(firstname, lastname){
+    console.log(firstname + ' ' + lastname);
+}
+const fullName1 = getFullName('Benjamin', 'Hughes'); //Benjamin Hughes
+const fullName2 = getFullName('Vie', 'Damayanti'); //Vie Damayanti
+
+//Function with an empty string as firstname and lastname.
+function getFullName(firstname, lastname){
+    if(firstname == "" && lastname == ""){
+        console.log('Please fill up your name');
+    }else{
+        console.log(firstname + ' ' + lastname)
+    }
+}    
+const fullName3 = getFullName('Brian', 'Jensen'); //Brian Jensen
+const fullName4 = getFullName("",""); //Please fill up your name
+
+//Event application
+function getEventWeekday(day){
+    const date = new Date();
+    const dayIndex = date.getDay();
+    const daysName = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    const eventIndex = (dayIndex + day) % 6;
+    console.log(daysName[eventIndex]);
+}
+getEventWeekday(5); //Wednesday
+
+//Weather wear
+const temperature = 20;
+function youCreateThisFunctionName(studentName){
+    if(temperature < 0 && temperature > 5){
+        console.log('You need to wear extra jacket')
+    } else if(temperature > 5 && temperature < 16){
+        console.log('Spring jacket is enough')
+    } else if( temperature > 16 && temperature < 23){
+        console.log('Tank top and jeans are the best')
+    } else {
+        console.log('Its bikini time')
+    }     
+}
+youCreateThisFunctionName(18); //Tank top and jeans are the best
+
+//Student manager
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if(studentName == ""){
+        console.log('You cannot add an empty string to a class');
```

Comment:
Instead of using `console.log`, the function should have `return "You cannot add an empty string to a class";`